ui/packages/catalog-ui-search/src/main/webapp/component/query-template-sharing/query-template-sharing.view.js

```diff
@@ -192,11 +193,15 @@ module.exports = Marionette.LayoutView.extend({
             data: JSON.stringify(templatePerms),
             context: this,
             success: function(data) {
-                this.message('Success!', 'Sharing Settings Saved for Query Template', 'success');
+                this.message('Success!', `Sharing Settings Saved for ${this.model.get('name')}`, 'success');
```

Comment:
@mackncheesiest You mentioned to just silently success, right?

Reply:
Yeah, other areas of the UI basically just throw up a `LoadingView` and then remove that loading view once it succeeds